customer-service/src/main/java/io/eventuate/examples/tram/ordersandcustomers/customers/domain/ReservedCredit.java

```diff
@@ -0,0 +1,60 @@
+package io.eventuate.examples.tram.ordersandcustomers.customers.domain;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+import java.math.BigDecimal;
+
+@Table("reserved_credit")
+public class ReservedCredit {
```

Comment:
This should be called CreditReservation and the table credit_reservation(s)

Reply:
I will fix